.github/workflows/_deploy-hl7-notification-cdk.yml

```
@@ -0,0 +1,181 @@
+          repository: metriport/metriport-internal
+          ref: ${{ ((inputs.deploy_env == 'production') && 'master') || 'develop' }}
+          token: ${{ secrets.SERVICE_PAT }} # secret token from user 'metriport-service'
+          path: metriport-internal
+          sparse-checkout: |
+            config-oss/api-infra
+
+      # checkout the internal repo to get configs - feature branch when "branching to staging"
+      - name: Determine which internal branch to checkout
+        env:
+          GITHUB_TOKEN: ${{ secrets.SERVICE_PAT }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:10:44: Double quote to prevent globbing and word splitting

(shellcheck)